portal-web/test/functional/com/liferay/portalweb/paths/portlet/frontenddataset/DataSet.path

```diff
@@ -340,6 +355,31 @@
 	<td>//div[@class="form-group"]//input[contains(@aria-label,'${key_label}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>ITEM_CHECKBOX_CHECKED</td>
+	<td>//button[contains(@class,'dropdown-item') and contains(.,'${key_name}')]//input[@type='checkbox' and @checked]</td>
+	<td></td>
+</tr>
```

Comment:
This already exists in Poshi. Please check Checkbox.path file.

Reply:
The checkbox is very specific to FDS filters, so it was necessary to create this path, which was only found in other components such as Object, Commerce, etc.

Reply:
what do you think?